### ISSUE TITLE
execute SQL queries (and store results in a list variable)

### ISSUE THREAD

**Issue by Njol** [Njol/Skript#223]

**Comment by saki2fifty** [comment]
Yes, this would be great!  If this is implemented, then we can pull data from any Plugin or Mod!  Can't wait for this...

**Comment by selfservice0** [comment]
bump for this.

**Comment by Njol** [comment]
The main issue right now is that the whole server would halt for a noticeable time if the database server is a remote server. I could create an effect 'execute SQL "<SQL here>"' and an event 'on SQL complete', but that would disconnect the querying trigger from the receiving trigger. Alternatively I could just stop the trigger while the SQL statement is being processed and resume once it's done (i.e. like the "wait" effect, but with an indeterminate time to wait), but that could confuse people, especially since it might seem like the trigger isn't interrupted, but it actually is.
